src/components/content/heroes/aurora/weather-api.ts

```diff
@@ -1,17 +1,65 @@
 import axios from 'axios';
 
 const API_KEY = process.env.WEATHER_API_KEY;
-const latitude = 48.1351;
-const longitude = 11.582;
+const BASE_URL = 'https://api.weatherapi.com/v1';
 
 export async function getWeather() {
-  const apiUrl = `https://api.weatherapi.com/v1/current.json?key=${API_KEY}&q=${latitude},${longitude}`;
+  try {
+    const response = await axios.get(
+      `${BASE_URL}/ip.json?key=${API_KEY}&q=auto:ip`,
+    );
+    const { lat, lon } = response.data;
+    return getWeatherOfLocation(lat, lon);
+  } catch (error) {
+    console.error('Error fetching IP geolocation:', error);
+    return 'NotSet';
```

Comment:
What about using `WeatherType.NotSet` here?
I think it should be 'Not set' instead of 'NotSet' and by using the enum directly we dont have to deal with these problems